src/QtImageReader.cpp

```diff
@@ -51,9 +60,65 @@ void QtImageReader::Open()
 	// Open reader if not already open
 	if (!is_open)
 	{
+
+#if USE_RESVG == 1
+		image = std::shared_ptr<QImage>(new QImage());
+		bool success;
+
+		QFileInfo image_file_info(QString::fromStdString(path));
+
+		// Only use resvg for files with a '.svg' extension
+		if (image_file_info.suffix() == "svg") {
+			// Init resvg structs
+			resvg_render_tree *rtree = nullptr;
+			resvg_options opt;
+			resvg_init_options(&opt);
+
+			// Set resvg options (ones where the defaults aren't what we want)
+			opt.dpi = 144;
```

Comment:
Why?

Reply:
I did not want to force the use of resvg as it was getting some push back from other developers, so I made it a compile-time option.  You can let the build system determine if resvg is available and use it if it is found, or you can override it and disable resvg outright.

Reply:
No, I'm about the DPI. Why 144?

Reply:
Oh, sorry 😁 

I don't recall exactly.  I do remember specifically wanting to set it, but I don't recall why at the moment.  (Should have included a comment)

Likely it was due to having too low of a DPI by default (lower than the DPI used elsewhere in OpenShot or libopenshot), at the time, or perhaps for antialiasing to get a smoother, less pixel-perfect rendering.